CONTRIBUTING.md

```diff
@@ -1,35 +1,211 @@
+
+
+*   **Explain which behavior you expected to see instead** and why;
+
+
+*   Use **screenshots/animated GIFs to help describe the issue** whenever appropriate [(How?)](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests);
+
+*   If the problem wasn't triggered by a specific action, describe what you were doing before the problem happened;
+
+*   **If you are reporting that qBittorrent crashes**, include the stack trace in the report; include it in a code block, a file attachment, or put it in a gist and provide link to that gist;
+
+*   **For performance-related issues**, include as much profiling data as you can (resource usage graphs, etc);
+
+*   Paste the **qBittorrent log** (or put the contents of the log in a gist and provide a link to the gist). The log can be viewed in the GUI (View -> Log -> tick all boxes). If you can't do that, the file is at:
+    -   Linux: `~/.local/share/data/qBittorrent/logs/qBittorrent.log`
+    -   Windows: `%LocalAppData%\qBittorrent\logs`
+    -   macOS: `~/Library/Application Support/qBittorrent/qBittorrent.log`
+
+
+-   **Do NOT post comments like "+1" or "me too!"** without providing new relevant info on the issue. Using the built-in reactions is OK though.
+
+-   If there seems to be an **issue with specific torrent files/magnet links**:
+    -   Always include links to the specific torrent(s)/magnet(s) where the issue occurs, even if "it happens everywhere".
+    -   Make sure you can't the problem with another client, to rule out the possibility that the issue is with the torrent file/magnet link itself.
+
+
+-   A screenshot, transcription or file upload of any of **qBittorrent's preferences that differ from the defaults**. Please include everything different from the defaults whether or not it seems relevant to your issue.
+
+-   **Attachment rules**:
+    -   Short logs and error messages can be pasted as quotes/code whenever small enough; otherwise make a gist with the contents and post the link to the gist;
+    -   Avoid linking/attaching impractical file formats such as PDFs/Word documents with images. If you want to post an image, just post the image.
+
+
+
+### Provide more context by answering these questions (if applicable):
+
+-   Are you **using qBittorrent with multiple monitors?** If so, can you reproduce the problem when you use a single monitor?
+
+-   Can you **reliably reproduce the issue?** If not, provide details about how often the problem happens and under which conditions it normally happens (e.g. only happens with extremely large torrents/only happens after qBittorrent is open for more than 2 days/etc...)
+
+-   Did the problem start happening recently (e.g. after updating to a new version of qBittorrent) or was this always a problem?
+
+-   If the problem started happening recently, can you reproduce the problem in an older version of qBittorrent? What's the most recent version in which the problem doesn't happen?
```

Comment:
The last sentence is redundant. If a user knows that he'll say it, it's common sense.

Reply:
Ok

Reply:
I disagree; I think we should be explicit in asking for this. Providing this information may not be as obvious to non-developers and general users.

In the best case, the user reads a superfluous sentence. In the worst, the issue requires a follow up (by one of us) to get this info.

Reply:
@Chocobo1 @thalieht ?

Reply:
Maybe @Piccirello is right but i have seen many issues that provide this information without being asked.